Demo/Sources/Model/Media.swift

```diff
@@ -22,13 +25,15 @@ struct Media: Hashable {
     let title: String
     let description: String?
     let image: URL?
+    let uiImage: UIImage?
```

Comment:
Misleading IMHO. I would rather call these properties `imageURL` and `image` respectively.